frontend/src/component/onboarding/sdkSnippets.ts

```diff
@@ -20,12 +20,15 @@ dotnet add package Newtonsoft.Json`,
     Vue: 'npm install @unleash/proxy-client-vue',
     Svelte: 'npm install @unleash/proxy-client-svelte',
     Swift: 'https://github.com/Unleash/unleash-proxy-client-swift',
-    Android:
-        'implementation("io.getunleash:unleash-android:${unleash.sdk.version}")',
+    Android: `implementation("io.getunleash:unleash-android:\${unleash.sdk.version}")
+
+// enabled required permissions
```

Comment:
android has an extra step compared to all other SDKs. We can either make it separate in the UI or add to the install section